emu_sv/sv_config.py

```diff
@@ -4,12 +4,20 @@
     CorrelationMatrix,
     QubitDensity,
     Fidelity,
+    Energy,
 )
+
+
 from emu_base import BackendConfig
 from emu_sv import StateVector
 from typing import Any
 
 
+from types import MethodType
+
+from emu_sv.custom_applies import custom_qubit_density
```

Comment:
I would rename `custom_applies` to `custom_callback_implementations`, that is more explicit